### DISCUSSION TITLE
Logging when message produced from Kafka-UI

### DISCUSSION THREAD

**Discussion by rjames4558** [provectus/kafka-ui#3217] [Q&A] [answered]
Is there any logging or other audit trail mechanism, that can be used to track which authenticated end user took action in Kafka-UI to produce a certain message ?

**Comment by Haarolean** [comment, marked as answer]
Hey,

not yet, unfortunately. It looks like #2998, but not in our nearest future plans, unfortunately.

Alternatively, you can enable [RBAC](https://provectus.gitbook.io/kafka-ui/configuration/rbac-role-based-access-control)
